src/lib/in.c

```diff
@@ -1768,6 +1742,12 @@ build_cflow_automaton(inputctx* ictx){
     { "[E", simple_cb_begin, },
     { "[<\\N;\\N;\\NM", mouse_press_cb, },
     { "[<\\N;\\N;\\Nm", mouse_release_cb, },
+    { "[<\\N;-\\N;\\NM", mouse_press_cb, },
```

Comment:
who's sending these?

Reply:
From a quick test it seems kitty, ghostty, xterm and urxvt do. alacritty, wezterm, gnome-terminal don't.